### PR TITLE
refactor(website/roadmap): update roadmap page

### PR DESCRIPTION
**Pull request by caqueste**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

**Comment by caqueste** [comment]
Deployment link → https://roadmapupdate2-dot-demo-dot-mozaic-design-system-dtrp.ey.r.appspot.com

File: src/docs/Updates/Roadmap/index.mdx [hunk 1]
```diff
@@ -39,12 +37,20 @@ description: 'Discover what we are working on right now'
...
 - Price tag
 
 ### For next sprints
 
+**Add new patterns:**
+
 - Card pattern
 - Hero pattern
+- Password policies
+
```
Comment (tiloyi) [inline src/docs/Updates/Roadmap/index.mdx:50]: The implementation of the new fonts is missing.

File: src/docs/Updates/Roadmap/index.mdx [hunk 2]
```diff
@@ -39,12 +37,20 @@ description: 'Discover what we are working on right now'
 
 **Add new pattern:**
 
+- Breadcrumb
 - Price tag
```
Comment (nherwegh) [inline src/docs/Updates/Roadmap/index.mdx:41]: I think this is way too ambitious, we're already late on some patterns from sprint 3

File: src/docs/Updates/Roadmap/index.mdx [hunk 3]
```diff
@@ -10,26 +10,24 @@ description: 'Discover what we are working on right now'
...
-- Improve the versioning system of Mozaic
+- ~~Tabs~~
 
 ### Sprint 3 (Week 26 to 28)
 
 **Add new patterns:**
 
+- Add Modal pattern
+- Accordion
 - Tooltip
```
Comment (nherwegh) [inline src/docs/Updates/Roadmap/index.mdx:30]: put tooltips in sprint 4?

Comment (caqueste) [reply src/docs/Updates/Roadmap/index.mdx:30]: Yes, you're right

File: src/docs/Updates/Roadmap/index.mdx (hunk 2 above)
Comment (caqueste) [reply src/docs/Updates/Roadmap/index.mdx:41]: Yes, I postpone the breadcrumb to sprint 5

File: src/docs/Updates/Roadmap/index.mdx [hunk 4]
```diff
@@ -10,41 +10,56 @@ description: 'Discover what we are working on right now'
...
+- ~~Modal~~
+- ~~Accordion~~
+
+### Sprint 4 (Week 28 to 30)
+
+**Add new patterns:**
+
 - Tooltip
-- Breadcrumb
+- Price tag
```
Comment (nherwegh) [inline src/docs/Updates/Roadmap/index.mdx:36]: sure about that?

**Pull request merged**